app/src/main/res/layout/movie_details_custom_view.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<merge xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
@ifeins actually I can see it in the design without the `FrameLayout` (?) anyway thanks. not using it enough 😄